src/config/index.js

```diff
@@ -4,6 +4,8 @@ import { fileURLToPath } from 'node:url'
 
 const dirname = path.dirname(fileURLToPath(import.meta.url))
 
+const isProduction = process.env.NODE_ENV === 'production'
```

Comment:
Id also the others, `development` and `test` and update the config defaults for `isProduction`, `isDevelopment` and `isTest` to use these consts

Reply:
```
const isProduction = process.env.NODE_ENV === 'production'
const isTest = process.env.NODE_ENV === 'test'
const isDevelopment = process.env.NODE_ENV === 'development'
```

Reply:
```
  isProduction: {
    doc: 'If this application running in the production environment',
    format: Boolean,
    default: isProduction
  },
  isDevelopment: {
    doc: 'If this application running in the development environment',
    format: Boolean,
    default: isDevelopment
  },
  isTest: {
    doc: 'If this application running in the test environment',
    format: Boolean,
    default: isTest
  },
```